### PR TITLE
block 25 scams

### PR DESCRIPTION
**Pull request by dubstard**
Fake 1inch
```
    "1inch-apps.com",
    "app-1inch.finance",
    "app-1inch.win",
    "app1inch.net",
```

Fake Coinbase
```
    "accountsecure-coinbase.com",
    "coinbase-account.servehalflife.com",
    "verifycoinbases.com",
    "help-coinbase-contact.coinbaseserviceaccount.com",
```

Fake Airdrop
```
    "airdrop.rissel.duckdns.org",
```

Fake biswap
```
    "biswaps-trade.com",
```

Fake Metamask 
```
    "createmetamask.org.authenticationvalidation.org",
    "metamask.eip-1559.network",
```
   
Fake Lido
```
    "lido-finance.cf",
    "lido-finance.gq",
    "lido-finance.ml",
    "lido-finance.ga",
    "lido-finance.tk",
```

Fake Pancakeswap
```
    "pancakeswap.net",
    "pancakeswaps.cc",
```

Fake Sushiswap
```
    "sushiswap.trade",
```

Fake Uniswap
```
    "uniswap.cn",
    "eli5uniswap.com",
    "ppuniswap.org",
```

419 scam
```
    "urbancapitalexchange.com",
```

Fake Yugalabs
```   
    "yugasIabs.com",
```

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
`yugasIabs.com` -> `yugasiabs.com`

**Review by 409H — APPROVED** [review]
